cmd/testutil/json2yaml/json2yaml.go

```diff
@@ -0,0 +1,28 @@
+package main
```

Comment:
I recommend adding a ref in the commit message. i.e. https://github.com/openshift/custom-metrics-autoscaler-operator/commit/d13a2a74b423c67408c929f9a45855460a508df5